pages/scenario_selection_page.py

```diff
@@ -298,6 +311,65 @@ def scenario_selection_page():
                             "RO, FiT, ECO (only proportion of revenue raised ordinarily via electricity units), WHD (only proportion of revenue raised ordinarily via electricity customers),"
                             " AAHEDC, NCC"
                         )  # ECO and WHD tricky to explain because they are currently levied on both gas and electricity
+    else:  # if a predefined scenario is selected
+        scenario_info = scenarios[selected_scenario]
+        ashp_life_span = config.life_span_default["ashp"]
+        boiler_life_span = config.life_span_default["boiler"]
+        ashp_maintenance_cost = config.maintenance_costs_default["ashp"]
+        boiler_maintenance_cost = config.maintenance_costs_default["boiler"]
+        ashp_maintenance_frequency = 1.0
+        boiler_maintenance_frequency = 1.0
+        ashp_efficiency = (
+            3.0
+            if scenario_info["ashp_scop"] == "reference"
+            else 3.5 if scenario_info["ashp_scop"] == "high" else 2.5
+        )
+        boiler_efficiency = config.boiler_efficiency_default
+        ashp_purchased_with_loan = (
+            "Yes" if scenario_info["purchasing_with_loans"] else "No"
+        )
+        if ashp_purchased_with_loan == "Yes":
+            ashp_loan_interes_rate = scenario_info["loan_interest_rate"]
+        else:
+            ashp_loan_interes_rate = 0
+
+        ashp_subsidy_model = scenario_info["ashp_subsidy"]
+
+        wholesale_price_projection = scenario_info["wholesale_price_projection"]
+
+        levy_rebalancing = scenario_info["levy_rebalancing"]
+
+        ashp_annual_cost_reduction = scenario_info["ashp_annual_cost_decrease"]
+
+        with st.expander("⚙️ Scenario parameter values"):
+            st.markdown(
+                "Under the selected scenario, the following parameters are set as follows:"
+            )
+
+            st.markdown(
+                f"""
+
+                        | Parameter | Gas Boiler | ASHP |
+                        |-----------|------------|-------|
+                        | Lifespan (years) | {boiler_life_span} |{ashp_life_span} |
+                        | Annual maintenance cost (£) | {boiler_maintenance_cost} |{ashp_maintenance_cost} |
+                        | Frequency of maintenance per year | {boiler_maintenance_frequency} |{ashp_maintenance_frequency} |
+                        | Efficiency | {boiler_efficiency} |{ashp_efficiency} |
+                        | Heating system purchased with a loan? | - | {ashp_purchased_with_loan} |
+                        | Loan interest rate |  - |{ashp_loan_interes_rate} |
+                        | Subsidy model |  - |{ashp_subsidy_model} |
+                        | Annual cost decrease | 0% | {scenario_info["ashp_annual_cost_decrease"]*100}% |
```

Comment:
```suggestion
                        | Annual cost reduction in market price of heating system installation  | 0% | {scenario_info["ashp_annual_cost_decrease"]*100}% |
```

Implementing suggested change on slide Feedback - ‘Scenario selection’ in Sprint 4 deck.